Code/Framework/AzCore/Tests/Math/MathUtilsTests.cpp

```diff
@@ -146,32 +151,27 @@ namespace UnitTest
         TestDivideAndRoundUpIsCorrect<uint64_t>();
     }
 
-    
-    class RoundUpInvalidInputTestsFixture : public ScopedAllocatorSetupFixture
-    {
-    };
-
-    TEST_F(RoundUpInvalidInputTestsFixture, DividAndRoundUp_AlignmentZeroUint32_Assert)
```

Comment:
Recommend adding a comment in this file about intentionally not testing divide by zero due to resulting in undefined behavior. This way someone does not try adding these tests back in.

Reply:
Added a comment